src/main/java/frc/robot/subsystems/Gripper.java

```diff
@@ -34,6 +34,10 @@ public void closeGripper() {
         solenoid.set(DoubleSolenoid.Value.kReverse);
     }
 
+    public boolean sensorHasTarget() { //@todo add sensor
```

Comment:
add that sensor
DigitalInput (DIO in RobotMap) at channel 3, call it lightSensor
also put its value in the Gripper namespace